### PR TITLE
Update migration guide with subscription changes

### PR DESCRIPTION
**Pull request by angrykoala**
Adds migration guide with all the changes on subscriptions

### CODE REVIEW

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-graphql-207.surge.sh

Updated pages:
- [Migration to 6.0.0](https://neo4j-docs-graphql-207.surge.sh/graphql/6/migration)

File: modules/ROOT/pages/migration/index.adoc [hunk 1]
```diff
@@ -28,6 +28,55 @@ The minimum Node.js version required to run the Neo4j GraphQL Library is now 20.
 
 The minimum Neo4j version required to run the Neo4j GraphQL Library is now 5.0.0.
 
+
+=== Removed Non-CDC subscriptions
+
+Support for subscription engines other than `Neo4jGraphQLSubscriptionsCDCEngine` has been dropped. This means subscriptions only work on databases with Change-Data-Capture enabled. The following previously supported engines are no longer supported:
```
Comment (rsill-neo4j) [inline modules/ROOT/pages/migration/index.adoc]:
```suggestion
Support for subscription engines other than `Neo4jGraphQLSubscriptionsCDCEngine` has been dropped.
This means subscriptions only work on databases with Change Data Capture (CDC) enabled. The following engines are no longer supported:
```

File: modules/ROOT/pages/migration/index.adoc [hunk 2]
```diff
@@ -28,6 +28,55 @@ The minimum Node.js version required to run the Neo4j GraphQL Library is now 20.
...
+new Neo4jGraphQL({
+    typeDefs,
+    driver,
+    features: {
+        subscriptions: true
+    },
+});
+----
+
+This means the default subscriptions now require CDC enabled in your database
```
Comment (rsill-neo4j) [inline modules/ROOT/pages/migration/index.adoc]:
```suggestion
This means the default subscriptions now require CDC enabled in your database.
```

**Comment by neo-technology-commit-status-publisher** [comment]
Thanks for the documentation updates.

The preview documentation has now been torn down - reopening this PR will republish it.

**Pull request merged**